tests/unit/amo/components/TestCollectionManager.js

```diff
@@ -256,6 +256,16 @@ describe(__filename, () => {
     }));
   });
 
+  it('disables submit button when the name is blank', () => {
+    const root = render();
+
+    // Enter in a blank collection name.
+    typeInput({ root, name: 'name', text: '' });
+
+    expect(root.find('.CollectionManager-submit'))
```

Comment:
Might be nice to add an extra `expect` statement here that makes sure the cancel button *isn't* disabled.